test/e2e/page-objects/pages/confirmations/redesign/transaction-confirmation.ts

```diff
@@ -79,13 +79,84 @@ class TransactionConfirmation extends Confirmation {
     text: tEn('review') as string,
   };
 
+  private readonly estimatedSimulationDetails = (type: string) => {
+    const css = '[data-testid="simulation-details-layout"]';
+    if (type === '') {
+      return css;
+    }
+
+    return {
+      css,
+      text: type,
+    };
+  };
+
+  private readonly outgoingIncomingSimulationDetails = (
+    isOutgoing: boolean,
+    index: number,
+  ) => {
+    const listTestId = isOutgoing
+      ? 'simulation-rows-outgoing'
+      : 'simulation-rows-incoming';
+    const id = index + 1;
+    const css = `[data-testid="${listTestId}"] [data-testid="simulation-details-balance-change-row"]:nth-child(${id})`;
+    console.log(`Locator for outgoing/incoming simulation details: ${css}`);
+
+    return css;
+  };
+
+  private readonly dappNumberConnected = (dappNumber: string) =>
+    By.xpath(`//p[normalize-space(.)='${dappNumber}']`);
+
   constructor(driver: Driver) {
     super(driver);
     this.driver = driver;
   }
 
-  private readonly dappNumberConnected = (dappNumber: string) =>
-    By.xpath(`//p[normalize-space(.)='${dappNumber}']`);
+  async expectBalanceChange(
+    isOutgoing: boolean,
+    index: number,
+    displayAmount: string,
+    assetName: string,
+  ) {
```

Comment:
just a small nit: if we had these arguments grouped as an object, that would improve a lot readability, whenever we call this function.

So then, instead of this:
```
 await transactionConfirmation.expectBalanceChange(
  true,
  0,
  '- 0.002',
  'ETH',
 );
 ```
We could have this:
```
 await transactionConfirmation.expectBalanceChange({
  isOutgoing: true,
  index: 0,
  displayAmount: '- 0.002',
  assetName: 'ETH',
});
 ```



```suggestion
  async expectBalanceChange({
    isOutgoing: boolean,
    index: number,
    displayAmount: string,
    assetName: string,
    }
```
```suggestion
  async expectBalanceChange({
    isOutgoing: boolean,
    index: number,
    displayAmount: string,
    assetName: string,
  }) {
```